ADAL/src/cache/ADTokenCache.m

```diff
@@ -69,6 +69,45 @@ - (void)dealloc
     SAFE_ARC_SUPER_DEALLOC();
 }
 
+- (id)cacheObjectForKey:(id<NSCopying>)key
+{
+    id object = nil;
+    @synchronized(self)
```

Comment:
Never mind. It's only called in that method. So, `getItemsImplKey:userId:` doesn't require a lock in the method itself.